src/main/app/claims/models/settlement.ts

```diff
@@ -40,6 +40,17 @@ export class Settlement {
     return partyStatement ? partyStatement.offer : undefined
   }
 
+  isLastOfferMadeByCourt (): boolean {
```

Comment:
I don't particularly like this - we duplicate the logic. Maybe I change and return full part statement but this will require a bit of refactoring. Or I introduce a new method to return higher object.